Doc/library/shelve.rst

```diff
@@ -149,8 +165,12 @@ Restrictions
       :const:`pickle.DEFAULT_PROTOCOL` is now used as the default pickle
       protocol.
 
+   .. versionchanged:: 3.14
+      Accepts custom *serializer* and *deserializer* functions in place of
+      :func:`pickle.dumps` and :func:`pickle.loads`.
```

Comment:
```suggestion
      Added the *serializer* and *deserializer* parameters.
```

I think you should not need to duplicate what you already said in `shelve.open`.